--- a/.vite/build/workspace-root-drop-handler-D1HnRjoH.js
+++ b/.vite/build/workspace-root-drop-handler-D1HnRjoH.js
@@ -39485,6 +39485,7 @@
     i = e.y(n.version);
   return (
     e.r({
+      enabled: !1,
       dsn: e.v,
       environment: vA.buildFlavor,
       release: i,
--- a/.vite/build/worker.js
+++ b/.vite/build/worker.js
@@ -66670,6 +66670,7 @@
   let n = kU(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (DA({
+    enabled: !1,
     dsn: jK,
     environment: e.buildFlavor,
     release: MK(n.version),
