--- a/.vite/build/worker.js
+++ b/.vite/build/worker.js
@@ -76002,6 +76002,7 @@
   let n = $V(e.appVersion),
     r = e.buildFlavor !== `prod`;
   (FD({
+    enabled: !1,
     dsn: sW,
     environment: e.buildFlavor,
     release: cW(n.version),
--- a/.vite/build/sqlite-Bd1pmlws.js
+++ b/.vite/build/sqlite-Bd1pmlws.js
@@ -42675,6 +42675,7 @@
     i = t.Yi(e.version);
   return (
     GB({
+      enabled: !1,
       dsn: t.Ji,
       environment: dV.buildFlavor,
       release: i,
