--- a/webview/assets/chunk-LFPYN7LY-CRkYzDF0.js
+++ b/webview/assets/chunk-LFPYN7LY-CRkYzDF0.js
@@ -1058,11 +1058,16 @@
   unstable_useTransitions: o,
 }) {
   let s = r.useRef();
-  s.current ??= a({ initialEntries: n, initialIndex: i, v5Compat: !0 });
+  s.current ??= a({
+    initialEntries: n ?? [window.__ELECTRON_SHIM__.initialRoute],
+    initialIndex: i,
+    v5Compat: !0,
+  });
   let c = s.current,
     [l, u] = r.useState({ action: c.action, location: c.location }),
     d = r.useCallback(
       (e) => {
+        window.__ELECTRON_SHIM__.onMemoryNavigationChanged(e);
         o === !1 ? u(e) : r.startTransition(() => u(e));
       },
       [o],
--- a/webview/assets/apps-Au-fp4y1.js
+++ b/webview/assets/apps-Au-fp4y1.js
@@ -2058,8 +2058,8 @@
   if (e == null) throw Error(`AppShellLayoutMotionContext is missing`);
   return e;
 }
-var Yi = j(M, !0),
-  Xi = j(M, () => new Jt(1)),
+var Yi = j(M, window.__ELECTRON_SHIM__.initialSidebarState),
+  Xi = j(M, () => new Jt(window.__ELECTRON_SHIM__.initialSidebarState)),
   Zi = j(F, !1),
   Qi = j(F, () => new Jt(0)),
   $i = j(F, !1),
